Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml

```
@@ -20,7 +20,7 @@
     shoes: ClothingShoesBootsCombatFilled
     head: ClothingHeadHatCentcom
     eyes: ClothingEyesGlassesSunglasses
-    gloves: ClothingHandsGlovesColorBlack
+    gloves: ClothingHandsGlovesColorBlackSecurity # WD
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)